install_config/aggregate_logging.adoc

```diff
@@ -96,14 +85,14 @@ $ oc project logging
 [NOTE]
 ====
 Specifying a non-empty
-link:../admin_guide/managing_projects.html#using-node-selectors[node
+xref:../admin_guide/managing_projects.html#using-node-selectors[node
 selector] on the project is not recommended, as this would restrict
 where Fluentd can be deployed. Instead, specify node selectors for the
-deployer to be applied on your other deployment configurations.
+deployer to be applied to your other deployment configurations.
 ====
 
-. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret
-can be configured specifically for your instance of the deployer.
+. Create a xref:../dev_guide/secrets.html#[secret] for the deployer. The secret
+can be configured specifically for the deployer.
```

Comment:
@sosiouxme OK. Now I get you. Changed to:

"Create a [secret] to provide security-related files to the deployer. While the secret is necessary, the contents of the secret are optional, and will be generated for you if none are supplied."